terraso_backend/apps/core/sites.py

```diff
@@ -0,0 +1,36 @@
+# Copyright © 2023 Technology Matters
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see https://www.gnu.org/licenses/.
+
+from django.conf import settings
+from django.contrib.sitemaps import GenericSitemap
+from django.contrib.sites.models import Site
+
+from apps.core.models import Landscape
+
+
+class TerrasoSitemap(GenericSitemap):
```

Comment:
Issue: This is the Web Client Sitemap, it should me named with that in mind, also this should not be part of the core app since this should be explicitly in a place just related to the webclient, also a comment should be added that ideally this code should not be here since the API should be agnostic of the existance of a web client